### PR TITLE
[Tests-Only][CI] Update .drone.star file for new changes from activity app

### PR DESCRIPTION
**Pull request by dpakach**
Related https://github.com/owncloud/QA/issues/678
- Fix code coverage reporting
- Implement early fail and full-ci in drone CI behavior like what is done in core and other repos

**note** this PR was generated by a automated script

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -985,13 +986,12 @@ def nonCoveragePipelines(ctx):
...
 def stagePipelines(ctx):
-	buildPipelines = build()
-	ldapIntegrationPipelines = ldapIntegration()
+	buildPipelines = build(ctx)
 	acceptancePipelines = acceptance(ctx)
-	if (buildPipelines == False) or (acceptancePipelines == False) or (ldapIntegrationPipelines == False):
+	if (buildPipelines == False) or (acceptancePipelines == False):
 		return False
 
-	return buildPipelines + ldapIntegrationPipelines + acceptancePipelines
```
Comment (phil-davis) [inline .drone.star:994]: There is special code in user_ldap. This needs to come back.

File: .drone.star [hunk 2]
```diff
@@ -2565,103 +2595,79 @@ def buildTestConfig(params):
 							configs.append(config)
 	return configs
 
-# This is custom starlark code added just for user_ldap
```
Comment (phil-davis) [inline .drone.star:2568]: This code needs to come back.

**Review by phil-davis — CHANGES_REQUESTED** [review]

**Comment by dpakach** [comment]
It gives error `starlark: maximum file size exceeded` now.
@phil-davis what do we do with this?

**Comment by phil-davis** [comment]
Reduce the number `numberOfParts` a bit - like I did in https://github.com/owncloud/encryption/pull/294/commits/6254f5528d871db51720e4323d25d03b30d1ff1b

You just have to try reducing bit-by-bit until it stops complaining!

**Comment by phil-davis** [comment]
@dpakach I can have a look at this. It also needs a commit like https://github.com/owncloud/encryption/pull/294/commits/813f0bd285a5d78aa9a6984c9a62121f0250978b

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>core-apiAll-21-16-master-mysql8.0-php7.2</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/user_ldap/3287/93/1

**Comment by phil-davis** [comment]
core PR https://github.com/owncloud/core/pull/38922 was merged today. We are running the very latest test code from core master, that has changed tests for public link upload. But `daily-master-qa` only has core code from last night. So we get fails, the same as https://github.com/owncloud/encryption/pull/294#issuecomment-874069329

This should pass tomorrow, when `daily-master-qa` has been updated overnight.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**